package.json

```diff
@@ -803,6 +805,17 @@
                 }
             }
         ],
+        "notebookProvider": [
+            {
+                "viewType": "julianotebook",
+                "displayName": "Julia Notebook",
+                "selector": [
+                    {
+                        "filenamePattern": "*.jlnb"
+                    }
+                ]
+            }
+        ],
```

Comment:
@DonJayamanne Is there something like a `notebookKernelProvider` here in `package.json`? I'd like to remove the whole `jlnb` file format stuff here, and I think also the `julianotebook` stuff, but then I'm wondering how we would get activated?

Reply:
Basically there's an event that there's fired when a notebook of a specific type opens, I'll send that info tomorrow, it's an entry in package.json (static activation).